.gitignore

```diff
@@ -93,7 +93,7 @@ target/
 
 # version information
 setup.cfg
-/src/bg-mpl-stylesheets/*/version.cfg
+/src/billingegroup/*/version.cfg
```

Comment:
this seems to be wrong?  was correct before?

Reply:
done